content/partner/azure.md

```diff
@@ -246,60 +232,15 @@ detail_sections:
           icon_color: yellow
           description: The Pulumi Azure Native provider is available in all Pulumi languages, including JavaScript, TypeScript, Python, Go, .NET, Java, and YAML. All SDKs are open source on GitHub and available as npm, NuGet, PyPI, and Go modules.
 
-superpowers:
-  - title: Multi Cloud
-    cta: Learn more
-    cta_url: "/docs/iac/get-started/azure/"
-    icon_type: cloud
-    description: |
-        Pulumi allows you to use top programming languages across all public clouds with support
-        for over 60 popular infrastructure service providers including private and hybrid clouds
-        helping ensure any multi-cloud strategy is successful.
-
-  - title: Reduce Provisioning Time
-    cta: Learn more
-    cta_url: "/docs/iac/get-started/azure/"
-    icon_type: provisioning
-    description: |
-        With Pulumi you are able to take advantage of the features of programming languages,
-        helping you reduce boilerplate code and ultimately ship Azure infrastructure and
-        applications faster with greater consistency.
-
-  - title: Automate Delivery
-    cta: Learn more
-    cta_url: "/docs/iac/packages-and-automation/continuous-delivery/"
-    icon_type: delivery
-    description: |
-        You can integrate Pulumi directly with your favorite CI/CD and SCM systems to
-        continuously deliver Azure infrastructure and applications. Improve the velocity
-        and visibility into your deployments from simple to complex global environments.
-
-  - title: Smart Architecture
-    cta: Learn more
-    cta_url: "/product/packages/"
-    icon_type: architecture
-    description: |
-        YAML and templated DSLs force you to write the same boilerplate code over and over.
-        Pulumi Packages allow you to codify those patterns and best practices so
-        you can stop reinventing the wheel and start inventing the platforms of the future.
-
-  - title: Be Proactive, Not Reactive
-    cta: Learn more
-    cta_url: "/docs/using-pulumi/crossguard/"
-    icon_type: policy
-    description: |
-        When you enable Pulumi's Policy as Code feature, you instantly gain the power to
-        prevent mistakes from being deployed. Enforce security, compliance, cost controls,
-        and best practices using policies defined in modern languages.
-
-  - title: Reduce Deployment Complexity
-    cta: Learn more
-    cta_url: "/docs/iac/concepts/testing/"
-    icon_type: testing
-    description: |
-        Deploying untested code can lead to some unexpected results. Pulumi lets you take advantage
-        of common tools, frameworks, and techniques to unit, integration, and property test your
-        Azure infrastructure. Ensure your infrastructure is correct before and after deployment.
+        - title: Built-in Guardrails
+          icon: code #This is supposed to be policy but I'm changing it to test something.
```

Comment:
You'll want to fix this. The build was blowing up because there's no `policy.html` in the `icons` folder.